### ISSUE TITLE
Fine-tuning

### ISSUE THREAD

**Issue by fiastros** [oke-aditya/image_similarity#4]
I'm working on cat facial recognition. so my first idea was to calculate the distance between two embedded cat faces. 
Is it possible to retrain/fine tune the embedding models for such task  ? 

Thank you.

**Comment by oke-aditya** [comment]
Yes of coure you can retrain these models on any dataset. The training scripts are provided here.
Note that this similarity search works on unlabelled data. It can help to compute distance between two cat faces.


But a few thoughts about cat facial recongition.
You can probably label the data and train a object detector such as FasterRCNN. These are available in torchvision. It would work better and faster.

**Comment by fiastros** [comment]
Thank you for your advice. 
I have tried yolov5 for object detection: it works fine but the issue is that it needs lots of images for new faces, and it needs retrain each time....
